src/package-operations.ts

```diff
@@ -36,6 +36,11 @@ export interface PackageManifest
   readonly [FieldNames.Workspaces]?: string[];
 }
 
+export interface ValidatedPackageManifest extends PackageManifest {
```

Comment:
Shouldn't this be something more like "MonorepoPackageManifest"? These properties might be absent on a valid manifest for a non-monorepo.

Reply:
Done in: b46666d